### PR TITLE
Refactor import dependency to es6

### PR DESCRIPTION
**Pull request by timse**
**What kind of change does this PR introduce?**

refactor

**Did you add tests for your changes?**

current tests should pass

**If relevant, link to documentation update:**

N/A

**Summary**

upgrade ImportDependency to es6

**Does this PR introduce a breaking change?**

No

### CODE REVIEW

File: lib/dependencies/ImportDependency.js [hunk 1]
```diff
@@ -2,32 +2,49 @@
...
+		if(!pathinfo) {
+			return "";
+		}
+
+		return `/*! ${shortenedRequest} */ `;
+	}
+
+	getContent(promise, dep, comment) {
+		if(dep.module) {
+			const prefix = promise ? promise : "Promise.resolve()";
```
Comment (sokra) [inline lib/dependencies/ImportDependency.js]: This increases bundle size compared to the original version. Please stay close to the original version when refactoring to ES6 and don't introduce functional changes.

Comment (timse) [inline lib/dependencies/ImportDependency.js]: i did not think of that! will revert the last commit!

**Comment by timse** [comment]
is the revert ok or should i force push it away :)?

File: lib/dependencies/ImportDependency.js (hunk 1 above)
Comment (TheLarkInn) [inline lib/dependencies/ImportDependency.js]: Revert or force push is fine

Comment (timse) [inline lib/dependencies/ImportDependency.js]: aye :) then I leave it as is

**Pull request merged**